ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -579,6 +697,7 @@ private void shutdown() {
       putManager.close();
       getManager.close();
       deleteManager.close();
+      ttlUpdateManager.close();
```

Comment:
Any reason for preference? It seems to me like the "source" (getmanager) needs to be cut off first (probably doesn't matter much in this case since the ops are quite short)